chart/kubeapps/Chart.yaml

```diff
@@ -1,19 +1,19 @@
 apiVersion: v1
 name: kubeapps
-version: 0.3.2
+version: 0.3.3
```

Comment:
should it be a minor version bump since MongoDB is a minor version bump?

Reply:
I don't know the criteria in charts but for example with npm modules if a dependency gets updated it doesn't affect to the main version. Following the semver notation this is a "bug fix" so it should be a patch version bump.